__tests__/NCN.test.js

```diff
@@ -131,33 +131,91 @@ describe('GET /api/articles', () => {
         .expect(200)
         .then((response) => {
             const body = response.body
-            body.forEach((article) => {
-                expect.objectContaining({
-                    author: expect.any(String),
-                    title: expect.any(String),
-                    article_id: expect.any(Number),
-                    topic: expect.any(String),
-                    created_at: expect.any(String),
-                    votes: expect.any(Number),
-                    article_img_url: expect.any(String),
-                    comment_count: expect.any(Number),
+            if( body.length > 0){
```

Comment:
The problem with checking like this is if the length is less than 0 once again this test will just pass.
we'll need to expect the length to be a number or expect the length not to be 0